apps/emqx_management/src/emqx_mgmt_api.erl

```diff
@@ -122,7 +159,7 @@ cluster_query(Params, Tab, QsSchema, QueryFun) ->
     Rows = do_cluster_query(Nodes, Tab, Qs, QueryFun, Start, Limit+1, []),
     Meta = #{page => Page, limit => Limit},
     NMeta = case CodCnt =:= 0 of
-                true -> Meta#{count => count(Tab, Nodes)};
+                true -> Meta#{count => lists:sum([rpc_call(Node, ets, info, [Tab, size], 5000) || Node <- Nodes])};
```

Comment:
lists:sum will crash on `badrpc`

Reply:
Maybe it's the right thing to let it crash

Reply:
then it results in a 500.
should be 503 with a sane error message.

Reply:
but we can fix it in another PR.